packages/transaction-controller/src/utils/feature-flags.ts

```diff
@@ -8,6 +8,22 @@ export const FEATURE_FLAG_TRANSACTIONS = 'confirmations_transactions';
 export const FEATURE_FLAG_EIP_7702 = 'confirmations_eip_7702';
 
 const DEFAULT_BATCH_SIZE_LIMIT = 10;
+const DEFAULT_GAS_ESTIMATE_FALLBACK_BLOCK_PERCENT = 35;
+
+type GasEstimateFallback = {
+  /**
+   * The fallback gas estimate for a transaction.
+   * This value is either a fixed hexadecimal number or a percentage multiplier.
+   */
+  value: number | Hex;
+
+  /**
+   * Indicates whether the gas estimate is a fixed hexadecimal number or a percentage multiplier.
+   * - `true`: The gas estimate is a fixed hexadecimal number.
+   * - `false`: The gas estimate is a percentage multiplier.
+   */
+  isFixedGas: boolean;
```

Comment:
Would it be cleaner and more explicit to just have two alternate optional properties such as `percentage` and `fixed`? We prioritise `fixed` if provided as it's the more precise.

All feature flag properties need to be optional since we can't validate externally, so need fallbacks internally for everything.